homeassistant/components/acaia/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "domain": "acaia",
+  "name": "Acaia",
```

Comment:
How do you pronounce this. My head wants to do it like acacia, but there's a c missing

Reply:
No idea tbh